src/JSHandle.ts

```diff
@@ -442,10 +464,11 @@ export class ElementHandle extends JSHandle {
    * @return {!Promise<!Array<!ElementHandle>>}
    */
   async $$(selector: string): Promise<ElementHandle[]> {
-    const arrayHandle = await this.evaluateHandle(
-        (element, selector) => element.querySelectorAll(selector),
-        selector
-    );
+    const defaultHandler = (element: Element, selector: string): ElementHandle[] => element.querySelectorAll(selector) as unknown as ElementHandle[];
+    const {updatedSelector, queryHandler} = this._getQueryHandlerAndSelector(selector, defaultHandler,
+        '$$ query set to use "QUERY_HANDLER_NAME", but no query function of that name was found');
```

Comment:
Looks like the error message is always the same. We could either parameterize the function name (`$`, `$$`, or `$$eval`) or just omit it. I think omitting seems fine:

    'Query set to use "QUERY_HANDLER_NAME", but no query function of that name was found'

That way we no longer need the `errorMessage` parameter.

Reply:
Done